rosdep/python.yaml

```diff
@@ -9411,6 +9411,19 @@ python3-rdflib:
   rhel:
     '8': ['python%{python3_pkgversion}-rdflib']
   ubuntu: [python3-rdflib]
+python3-reduct-py-pip:
+  debian:
```

Comment:
Just a drive-by comment here, mainly for @nuclearsandwich . Is there a reason we didn't do the following wildcard expansion?

```yaml
python3-reduct-py-pip:
  '*':
    pip:
      packages: [reduct-py]
```

Reply:
OS level wildcards were only added recently (well, 2022 apparently but in ROS years that's still puppy age) https://github.com/ros-infrastructure/rosdep/blob/master/CHANGELOG.rst#0220-2022-06-24 and it doesn't always occur to suggest them as a result. It's a good change to suggest here.